requirements-ftest.txt

```diff
@@ -1,6 +1,6 @@
-avocado-framework==82
-avocado-framework-plugin-result-html==82
-avocado-framework-plugin-varianter-yaml-to-mux==82
+avocado-framework>=103.0,<104.0
```

Comment:
This range ensures we pick up any dot fixes to the 103 LTS release.

Reply:
Does `==103` not allow any 103.x point release?  That was my experience with `==82`.

Reply:
According to https://peps.python.org/pep-0440/#version-matching it looks like we could use `==103.*`.